patch/kernel/archive/rockchip64-5.10/linux-90112-add-rk3399-roc-pc-plus-dts.patch

```diff
@@ -0,0 +1,134 @@
+--- /dev/null
```

Comment:
Why totally replacing DT with another patch instead of modifying the existing one?